packages/showcase/README.md

```diff
@@ -2,6 +2,114 @@
 
 Creates a static website with documentation, API and demos for all components described in the meta files.
 
+## Routing and categorization
+
+the website route is created from categorization of assets.
+
+for instance a Date input component can be categorized as components/inputs/date/date-input.
+
+
+### categorization using the meta-data-store:
+
+you can categorize components using the meta-data-repo:
+
+```ts
+import Registry from '@ui-autotools/registry';
+import {Composite} from './composite';
+import style1 from './variant1.st.css';
+
+const metadata = Registry.getComponentMetadata(Composite);
+
+metadata.categorize('components/inputs/date/date-input');
```

Comment:
I'm worried about when we'd want to do some tree restructuring like adding additional level or even just renaming of directories. Maybe if we just reference to a parent directory it will be simpler to manage in site settings